cmd/gardener-apiserver/app/gardener_apiserver.go

```diff
@@ -375,9 +376,59 @@ func (o *Options) Run(ctx context.Context) error {
 		return err
 	}
 
+	if err := server.GenericAPIServer.AddPostStartHook("bootstrap-public-info", func(_ genericapiserver.PostStartHookContext) error {
+		p := publicInfo{
+			Version:      version.Get().String(),
+			FeatureGates: fmt.Sprint(features.DefaultFeatureGate),
+		}
+
+		if len(o.ExtraOptions.WorkloadIdentityTokenIssuer) != 0 {
+			p.WorkloadIdentityIssuerURL = &o.ExtraOptions.WorkloadIdentityTokenIssuer
+		}
+
+		marsheledInfo, err := yaml.Marshal(p)
```

Comment:
Nit: Typo.

(Not sure if we use American English or British English per default. Hence, it could be `marshaledInfo` or `marshalledInfo`.)

```suggestion
		marshalledInfo, err := yaml.Marshal(p)
```